spec/truffle/pre_initialization_spec.rb

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+# OTHER DEALINGS IN THE SOFTWARE.
+
+require_relative '../ruby/spec_helper'
+
+guard -> { Truffle.native? } do
+  describe "The pre-initialized context" do
+    it "can be used to run specs, as otherwise this spec run is meaningless" do
```

Comment:
I don't really get how this spec title matches the body.

Reply:
If the pre-initialized context was not used (because of e.g. incompatible options), then all other specs would also run on a not-pre-initialized context which would be a big problem. So this checks that while running specs we are using the pre-initialized context.
Any idea how to clarify?